meta-luneos/recipes-multimedia/libpng/libpng_1.6.37.bb

```diff
@@ -2,15 +2,15 @@ SUMMARY = "PNG image format decoding library"
 HOMEPAGE = "http://www.libpng.org/"
 SECTION = "libs"
 LICENSE = "Libpng"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=67d8837410863f9821bbd606536f0329 \
-                    file://png.h;endline=144;md5=abfa0497feb393b5842d3d82c1009520"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b0085051bf265bac2bfc38bc89f50000 \
+                    file://png.h;endline=263;md5=d7f847120588784a6d62ff5970725e89"
 DEPENDS = "zlib"
```

Comment:
1.36 is already in oe-core warrior we're using and 1.37 is in zeus, we should drop this from meta-lune*

The only difference with zeus is this:

docker-shr @ ~/openembedded-core $ diff meta/recipes-multimedia/libpng/libpng_1.6.37.bb ~/meta-webos-ports/meta-luneos/recipes-multimedia/libpng/libpng_1.6.37.bb
5,7c5,6
< LIC_FILES_CHKSUM = "file://LICENSE;md5=b0085051bf265bac2bfc38bc89f50000\
<                     file://png.h;endline=144;md5=8acd23d544623816b097e07be0139509\
<                     "
---
> LIC_FILES_CHKSUM = "file://LICENSE;md5=b0085051bf265bac2bfc38bc89f50000 \
>                     file://png.h;endline=263;md5=d7f847120588784a6d62ff5970725e89"
17,18d15
<
< UPSTREAM_CHECK_URI = "http://libpng.org/pub/png/libpng.html"